forge-gui/res/cardsfolder/upcoming/mesmerizing_dose.txt

```diff
@@ -2,7 +2,7 @@ Name:Mesmerizing Dose
 ManaCost:1 U U
 Types:Enchantment Aura
 K:Enchant creature
-A:SP$ Attach | Cost$ 1 B | ValidTgts$ Creature.YouCtrl | AITgts$ Card.nonToken | AILogic$ Pump
+A:SP$ Attach | Cost$ 1 U U | ValidTgts$ Creature
```

Comment:
Removing something isn't always the right fix, so it's best to check if unsure
in this case the AI would no longer play it
you should use `AILogic$ KeepTapped`

Reply:
Thanks for the explanation will keep in mind